src/interfaces.js

```diff
@@ -132,5 +132,5 @@ interface ConfigureRouter {
   /**
   * Implement this hook if you want to add sub-routes to your view.
   */
-  configureRouter(config: RouterConfiguration, router: Router);
+  configureRouter(config: RouterConfiguration, router: Router): any;
```

Comment:
> Also in your sample the configureRouter is missing explicit return type. 

Yes, i also discovered that myself, when writing my previous comment - as I said,
>By the way, it is the only method in my sample that isn't written by myself, but my teammate.

so it slipped through my code review.

But I'd still like to get a reply from the Aurelia team:
> @EisenbergEffect, @bryanrsmith - could You clarify the return type of configureRouter?